packages/calcite-components/src/components/navigation-user/navigation-user.scss

```diff
@@ -30,24 +26,26 @@
       font-sans;
     background-color: var(--calcite-navigation-user-background-color, transparent);
     border: none;
-    border-block-end: var(--calcite-spacing-base) solid var(--calcite-navigation-user-border-color, transparent);
+    border-block-end-width: var(--calcite-spacing-base);
+    border-block-end-style: solid;
+    border-color: var(--calcite-navigation-user-border-color, transparent);
     font-size: var(--calcite-font-size-md);
     line-height: var(--calcite-font-line-height-fixed-lg);
   }
 }
 
 :host(:hover) .button,
 :host(:focus) .button {
-  background-color: var(--calcite-navigation-user-background-color-hover, var(--calcite-color-foreground-2));
+  --calcite-navigation-user-background-color: var(--calcite-color-foreground-2);
```

Comment:
this should be `background-color: var(--calcite-navigation-user-background-color, var(--calcite-color-foreground-2));`